src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -54968,8 +54968,8 @@ private static string BuildAssemblyExternClause(PortableExecutableReference refe
         public void WindowsRuntimeEvent_01()
         {
             var ilSource =
-BuildAssemblyExternClause(TestMetadata.NetCoreApp31.SystemRuntime) +
-BuildAssemblyExternClause(TestMetadata.NetCoreApp31.SystemRuntimeInteropServicesWindowsRuntime) +
+BuildAssemblyExternClause(TestMetadata.Net451.SystemRuntime) +
+BuildAssemblyExternClause(TestMetadata.Net451.SystemRuntimeInteropServicesWindowsRuntime) +
```

Comment:
💭 I'm guessing this is moving back from .NET Core 3.1 to .NET Framework 4.5.1 because these two support UAP but .NET 5.0 does not?

Reply:
Correct. When I asked @AaronRobinsonMSFT about why these were removed there was a bit of surprise that we had taken any dependency on it. It's believed to be a non-customer scenario overall. My hope is we can keep the check in place using these assemblies but given its' a non-scenario I'd _consider_ just removing the test if it became burdensome to maintain.